### ISSUE TITLE
tp-link atcher c7 v2 reset button does not work 

### ISSUE THREAD

**Issue by kkwestt** [xchwarze/wifi-pineapple-cloner-builds#8]
Upgrade from OEM firmware to archer-c7-v2-universal-sysupgrade.bin
After boot i see Welcome page 
next Changes for 2.8.4 page
next Secure Setup

Disabling WiFi is recommended for initial setup.

Quickly press the reset button (underside of the WiFi Pineapple) to continue with WiFi disabled.

Hold the reset button for two or more seconds to continue with WiFi enabled.

Try push buttons and wifi adapters **nothing**.

log:
```

19:55:38.924 -> U-Boot 1.1.4 (Mar  8 2018 - 10:11:32)
19:55:38.924 -> 
19:55:38.924 -> ap135 - Scorpion 1.0DRAM:  
19:55:38.924 -> sri
19:55:38.924 -> Scorpion 1.0
19:55:38.924 -> ath_ddr_initial_config(179): (32bit) ddr2 init
19:55:38.924 -> tap = 0x00000003
19:55:38.924 -> Tap (low, high) = (0x3, 0x1c)
19:55:38.924 -> Tap values = (0xf, 0xf, 0xf, 0xf)
19:55:38.924 -> 128 MB
19:55:38.924 -> Flash Manuf Id 0x1, DeviceId0 0x20, DeviceId1 0x18
19:55:38.924 -> flash size 16MB, sector count = 256
19:55:38.924 -> Flash: 16 MB
19:55:38.924 -> Using default environment
19:55:38.924 -> 
19:55:39.413 -> *** Warning *** : PCIe WLAN Module not found !!!
19:55:39.413 -> In:    serial
19:55:39.413 -> Out:   serial
19:55:39.413 -> Err:   serial
19:55:39.413 -> Net:   ath_gmac_enet_initialize...
19:55:39.413 -> athrs_sgmii_res_cal: cal value = 0xe
19:55:39.413 -> No valid address in Flash. Using fixed address
19:55:39.413 -> No valid address in Flash. Using fixed address
19:55:39.768 -> ath_gmac_enet_initialize: reset mask:c02200 
19:55:39.768 -> Scorpion  ----> S17 PHY *
19:55:39.768 -> Vlan config...
19:55:40.299 -> TEST: FINAL REG VAL after TX Calibration - 0x86000000
19:55:40.299 -> TEST: FINAL XMII VAL after RX Calibration - 0x96000000
19:55:40.299 -> TEST: FINAL ETH_CFG VAL after RX Calibration - 0x00028001
19:55:40.299 -> athrs17_reg_init: complete
19:55:40.299 -> : cfg1 0x80000000 cfg2 0x7335
19:55:40.299 -> eth0: ba:be:fa:ce:08:41
19:55:41.065 -> eth0 up
19:55:41.065 -> athrs17_reg_init_wan done
19:55:41.065 -> SGMII in forced mode
19:55:41.065 -> athr_gmac_sgmii_setup SGMII done
19:55:41.065 -> : cfg1 0x800c0000 cfg2 0x7214
19:55:41.616 -> eth1: ba:be:fa:ce:08:41
19:55:42.623 -> eth1 up
19:55:42.623 -> eth0, eth1
19:55:42.623 -> Setting 0x18116290 to 0x58b1214f
19:55:42.623 -> Autobooting in 1 seconds
19:55:43.173 -> ## Booting image at 9f020000 ...
19:55:43.173 ->    Uncompressing Kernel Image ... OK
19:55:44.991 -> 
19:55:44.991 -> Starting kernel ...
19:55:44.991 -> 
19:55:44.991 -> [    0.000000] Linux version 4.14.221 (builder@buildhost) (gcc version 7.5.0 (OpenWrt GCC 7.5.0 r11306-c4a6851c72)) #0 Mon Feb 15 15:22:37 2021
19:55:45.398 -> [    0.000000] bootconsole [early0] enabled
19:55:45.398 -> [    0.000000] CPU0 revision is: 00019750 (MIPS 74Kc)
19:55:45.398 -> [    0.000000] SoC: Qualcomm Atheros QCA9558 ver 1 rev 0
19:55:45.398 -> [    0.000000] Determined physical RAM map:
19:55:45.398 -> [    0.000000]  memory: 08000000 @ 00000000 (usable)
19:55:45.398 -> [    0.000000] Initrd not found or empty - disabling initrd
19:55:45.398 -> [    0.000000] Primary instruction cache 64kB, VIPT, 4-way, linesize 32 bytes.
19:55:45.398 -> [    0.000000] Primary data cache 32kB, 4-way, VIPT, cache aliases, linesize 32 bytes
19:55:45.398 -> [    0.000000] Zone ranges:
19:55:45.398 -> [    0.000000]   Normal   [mem 0x0000000000000000-0x0000000007ffffff]
19:55:45.398 -> [    0.000000] Movable zone start for each node
19:55:45.398 -> [    0.000000] Early memory node ranges
19:55:45.398 -> [    0.000000]   node   0: [mem 0x0000000000000000-0x0000000007ffffff]
19:55:45.398 -> [    0.000000] Initmem setup node 0 [mem 0x0000000000000000-0x0000000007ffffff]
19:55:45.398 -> [    0.000000] random: get_random_bytes called from 0x804f0740 with crng_init=0
19:55:45.398 -> [    0.000000] Built 1 zonelists, mobility grouping on.  Total pages: 32512
19:55:45.398 -> [    0.000000] Kernel command line:  board=ARCHER-C7-V2  console=ttyS0,115200 rootfstype=squashfs noinitrd
19:55:45.398 -> [    0.000000] PID hash table entries: 512 (order: -1, 2048 bytes)
19:55:45.398 -> [    0.000000] Dentry cache hash table entries: 16384 (order: 4, 65536 bytes)
19:55:45.398 -> [    0.000000] Inode-cache hash table entries: 8192 (order: 3, 32768 bytes)
19:55:45.398 -> [    0.000000] Writing ErrCtl register=00000000
19:55:45.398 -> [    0.000000] Readback ErrCtl register=00000000
19:55:45.398 -> [    0.000000] Memory: 124292K/131072K available (3955K kernel code, 192K rwdata, 516K rodata, 320K init, 212K bss, 6780K reserved, 0K cma-reserved)
19:55:45.398 -> [    0.000000] SLUB: HWalign=32, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
19:55:45.398 -> [    0.000000] NR_IRQS: 51
19:55:45.398 -> [    0.000000] Clocks: CPU:720.000MHz, DDR:600.000MHz, AHB:200.000MHz, Ref:40.000MHz
19:55:45.398 -> [    0.000000] clocksource: MIPS: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 5309056796 ns
19:55:45.398 -> [    0.000007] sched_clock: 32 bits at 360MHz, resolution 2ns, wraps every 5965232126ns
19:55:45.398 -> [    0.008303] Calibrating delay loop... 358.80 BogoMIPS (lpj=1794048)
19:55:45.398 -> [    0.074984] pid_max: default: 32768 minimum: 301
19:55:45.398 -> [    0.080072] Mount-cache hash table entries: 1024 (order: 0, 4096 bytes)
19:55:45.398 -> [    0.087114] Mountpoint-cache hash table entries: 1024 (order: 0, 4096 bytes)
19:55:45.398 -> [    0.096377] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
19:55:45.398 -> [    0.106870] futex hash table entries: 256 (order: -1, 3072 bytes)
19:55:45.398 -> [    0.114299] NET: Registered protocol family 16
19:55:45.398 -> [    0.120120] MIPS: machine is TP-LINK Archer C7
19:55:45.398 -> [    0.127910] ar724x-pci ar724x-pci.0: PCIe link is down
19:55:45.398 -> [    0.133416] registering PCI controller with io_map_base unset
19:55:45.398 -> [    0.139647] registering PCI controller with io_map_base unset
19:55:45.838 -> [    0.636159] PCI host bridge to bus 0000:00
19:55:45.838 -> [    0.640525] pci_bus 0000:00: root bus resource [mem 0x10000000-0x11ffffff]
19:55:45.838 -> [    0.647859] pci_bus 0000:00: root bus resource [io  0x0000]
19:55:46.141 -> [    0.653782] pci_bus 0000:00: root bus resource [??? 0x00000000 flags 0x0]
19:55:46.141 -> [    0.661010] pci_bus 0000:00: No busn resource found for root bus, will use [bus 00-ff]
19:55:46.141 -> [    0.669558] PCI host bridge to bus 0000:01
19:55:46.141 -> [    0.673943] pci_bus 0000:01: root bus resource [mem 0x12000000-0x13ffffff]
19:55:46.141 -> [    0.681248] pci_bus 0000:01: root bus resource [io  0x0001]
19:55:46.141 -> [    0.687184] pci_bus 0000:01: root bus resource [??? 0x00000000 flags 0x0]
19:55:46.141 -> [    0.694403] pci_bus 0000:01: No busn resource found for root bus, will use [bus 01-ff]
19:55:46.141 -> [    0.703226] pci 0000:01:00.0: BAR 0: assigned [mem 0x12000000-0x121fffff 64bit]
19:55:46.141 -> [    0.711020] pci 0000:01:00.0: BAR 6: assigned [mem 0x12200000-0x1220ffff pref]
19:55:46.141 -> [    0.721518] clocksource: Switched to clocksource MIPS
19:55:46.141 -> [    0.727729] NET: Registered protocol family 2
19:55:46.141 -> [    0.733112] TCP established hash table entries: 1024 (order: 0, 4096 bytes)
19:55:46.141 -> [    0.740533] TCP bind hash table entries: 1024 (order: 0, 4096 bytes)
19:55:46.141 -> [    0.747342] TCP: Hash tables configured (established 1024 bind 1024)
19:55:46.141 -> [    0.754223] UDP hash table entries: 256 (order: 0, 4096 bytes)
19:55:46.141 -> [    0.760446] UDP-Lite hash table entries: 256 (order: 0, 4096 bytes)
19:55:46.141 -> [    0.767346] NET: Registered protocol family 1
19:55:46.141 -> [    0.774377] Crashlog allocated RAM at address 0x3f00000
19:55:46.141 -> [    0.780942] workingset: timestamp_bits=30 max_order=15 bucket_order=0
19:55:46.141 -> [    0.791923] squashfs: version 4.0 (2009/01/31) Phillip Lougher
19:55:46.141 -> [    0.798123] jffs2: version 2.2 (NAND) (SUMMARY) (LZMA) (RTIME) (CMODE_PRIORITY) (c) 2001-2006 Red Hat, Inc.
19:55:46.141 -> [    0.816368] io scheduler noop registered
19:55:46.141 -> [    0.820543] io scheduler deadline registered (default)
19:55:46.141 -> [    0.826402] Serial: 8250/16550 driver, 16 ports, IRQ sharing enabled
19:55:46.141 -> [    0.834888] console [ttyS0] disabled
19:55:46.141 -> [    0.858770] serial8250.0: ttyS0 at MMIO 0x18020000 (irq = 11, base_baud = 2500000) is a 16550A
19:55:46.141 -> [    0.867965] console [ttyS0] enabled
19:55:46.141 -> [    0.867965] console [ttyS0] enabled
19:55:46.141 -> [    0.875392] bootconsole [early0] disabled
19:55:46.141 -> [    0.875392] bootconsole [early0] disabled
19:55:46.141 -> [    0.889284] m25p80 spi0.0: found s25fl128s, expected m25p80
19:55:46.141 -> [    0.895017] m25p80 spi0.0: s25fl128s (16384 Kbytes)
19:55:46.141 -> [    0.900922] 5 tp-link partitions found on MTD device spi0.0
19:55:46.141 -> [    0.906607] Creating 5 MTD partitions on "spi0.0":
19:55:46.141 -> [    0.911472] 0x000000000000-0x000000020000 : "u-boot"
19:55:46.141 -> [    0.917599] 0x000000020000-0x0000001a4254 : "kernel"
19:55:46.141 -> [    0.924256] 0x0000001a4254-0x000000ff0000 : "rootfs"
19:55:46.141 -> [    0.929895] mtd: device 2 (rootfs) set to be root filesystem
19:55:46.141 -> [    0.935741] 1 squashfs-split partitions found on MTD device rootfs
19:55:46.141 -> [    0.942035] 0x000000d10000-0x000000ff0000 : "rootfs_data"
19:55:46.141 -> [    0.949345] 0x000000ff0000-0x000001000000 : "art"
19:55:46.609 -> [    0.955211] 0x000000020000-0x000000ff0000 : "firmware"
19:55:46.609 -> [    0.962808] libphy: Fixed MDIO Bus: probed
19:55:46.609 -> [    0.977036] switch0: Atheros AR8327 rev. 4 switch registered on ag71xx-mdio.0
19:55:46.885 -> [    1.672356] libphy: ag71xx_mdio: probed
19:55:47.510 -> [    2.303043] ag71xx ag71xx.0: connected to PHY at ag71xx-mdio.0:00 [uid=004dd034, driver=Atheros AR8216/AR8236/AR8316]
19:55:47.510 -> [    2.314283] eth0: Atheros AG71xx at 0xb9000000, irq 4, mode: rgmii
19:55:48.022 -> [    2.601516] random: fast init done
19:55:48.314 -> [    2.943045] eth1: Atheros AG71xx at 0xba000000, irq 5, mode: sgmii
19:55:48.314 -> [    2.951001] NET: Registered protocol family 10
19:55:48.314 -> [    2.959563] Segment Routing with IPv6
19:55:48.314 -> [    2.963403] NET: Registered protocol family 17
19:55:48.314 -> [    2.967952] bridge: filtering via arp/ip/ip6tables is no longer available by default. Update your scripts to load br_netfilter if you need this.
19:55:48.314 -> [    2.981580] 8021q: 802.1Q VLAN Support v1.8
19:55:48.314 -> [    2.987087] hctosys: unable to open rtc device (rtc0)
19:55:48.314 -> [    2.999138] VFS: Mounted root (squashfs filesystem) readonly on device 31:2.
19:55:48.314 -> [    3.007496] Freeing unused kernel memory: 320K
19:55:48.314 -> [    3.012035] This architecture does not have kernel memory protection.
19:55:49.098 -> [    3.817121] init: Console is alive
19:55:49.098 -> [    3.820749] init: - watchdog -
19:55:51.283 -> [    6.078130] kmodloader: loading kernel modules from /etc/modules-boot.d/*
19:55:51.695 -> [    6.233153] usbcore: registered new interface driver usbfs
19:55:51.695 -> [    6.238799] usbcore: registered new interface driver hub
19:55:51.695 -> [    6.244292] usbcore: registered new device driver usb
19:55:51.695 -> [    6.276572] ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
19:55:51.695 -> [    6.290050] SCSI subsystem initialized
19:55:51.695 -> [    6.297742] ehci-platform: EHCI generic platform driver
19:55:51.695 -> [    6.303173] ehci-platform ehci-platform.0: EHCI Host Controller
19:55:51.695 -> [    6.309205] ehci-platform ehci-platform.0: new USB bus registered, assigned bus number 1
19:55:51.695 -> [    6.319477] ehci-platform ehci-platform.0: TX-TX IDP fix enabled
19:55:51.695 -> [    6.325599] ehci-platform ehci-platform.0: irq 48, io mem 0x1b000000
19:55:51.695 -> [    6.361558] ehci-platform ehci-platform.0: USB 2.0 started, EHCI 1.00
19:55:51.695 -> [    6.368735] hub 1-0:1.0: USB hub found
19:55:51.695 -> [    6.372849] hub 1-0:1.0: 1 port detected
19:55:51.695 -> [    6.377226] ehci-platform ehci-platform.1: EHCI Host Controller
19:55:51.695 -> [    6.383299] ehci-platform ehci-platform.1: new USB bus registered, assigned bus number 2
19:55:51.695 -> [    6.393580] ehci-platform ehci-platform.1: TX-TX IDP fix enabled
19:55:51.695 -> [    6.399686] ehci-platform ehci-platform.1: irq 49, io mem 0x1b400000
19:55:51.695 -> [    6.431601] ehci-platform ehci-platform.1: USB 2.0 started, EHCI 1.00
19:55:51.695 -> [    6.438790] hub 2-0:1.0: USB hub found
19:55:51.695 -> [    6.442921] hub 2-0:1.0: 1 port detected
19:55:51.695 -> [    6.449762] ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
19:55:51.695 -> [    6.457192] ohci-platform: OHCI generic platform driver
19:55:51.695 -> [    6.464320] uhci_hcd: USB Universal Host Controller Interface driver
19:55:51.695 -> [    6.474321] usbcore: registered new interface driver usb-storage
19:55:51.695 -> [    6.480964] kmodloader: done loading kernel modules from /etc/modules-boot.d/*
19:55:51.695 -> [    6.490806] init: - preinit -
19:55:52.226 -> [    7.011587] usb 1-1: new high-speed USB device number 2 using ehci-platform
19:55:52.574 -> [    7.339734] eth1: link up (1000Mbps/Full duplex)
19:55:52.574 -> [    7.345684] random: procd: uninitialized urandom read (4 bytes read)
19:55:53.139 -> [    7.451717] mount_root: loading kmods from internal overlay
19:55:53.139 -> [    7.490613] kmodloader: loading kernel modules from //etc/modules-boot.d/*
19:55:53.139 -> [    7.501768] kmodloader: done loading kernel modules from //etc/modules-boot.d/*
19:55:53.581 -> [    8.383253] block: attempting to load /etc/config/fstab
19:55:53.581 -> [    8.392846] block: unable to load configuration (fstab: Entry not found)
19:55:53.918 -> [    8.399763] block: no usable configuration
19:55:53.918 -> [    8.404611] mount_root: jffs2 not ready yet, using temporary tmpfs overlay
19:55:53.918 -> [    8.452222] urandom-seed: Seed file not found (/etc/urandom.seed)
19:55:53.918 -> [    8.554752] eth1: link down
19:55:53.918 -> [    8.567168] procd: - early -
19:55:53.918 -> [    8.570172] procd: - watchdog -
19:55:54.322 -> [    9.117901] random: jshn: uninitialized urandom read (4 bytes read)
19:55:54.586 -> [    9.243779] procd: - watchdog -
19:55:54.586 -> [    9.247223] procd: - ubus -
19:55:54.833 -> [    9.461679] random: ubusd: uninitialized urandom read (4 bytes read)
19:55:54.833 -> [    9.480278] random: ubusd: uninitialized urandom read (4 bytes read)
19:55:54.833 -> [    9.491470] procd: - init -
19:55:54.988 -> Please press Enter to activate this console.
19:55:56.131 -> [   10.938091] kmodloader: loading kernel modules from /etc/modules.d/*
19:55:56.248 -> [   10.985977] Key type dns_resolver registered
19:55:56.248 -> [   11.041820] RPC: Registered named UNIX socket transport module.
19:55:56.248 -> [   11.047823] RPC: Registered udp transport module.
19:55:56.248 -> [   11.052629] RPC: Registered tcp transport module.
19:55:56.248 -> [   11.057396] RPC: Registered tcp NFSv4.1 backchannel transport module.
19:55:56.390 -> [   11.183770] ip6_tables: (C) 2000-2006 Netfilter Core Team
19:55:56.425 -> [   11.226924] Loading modules backported from Linux version v4.19.161-0-gdaefdc9eb24b
19:55:56.425 -> [   11.234733] Backport generated by backports.git v4.19.161-1-0-g4bb568fe
19:55:56.500 -> [   11.277759] ip_tables: (C) 2000-2006 Netfilter Core Team
19:55:56.500 -> [   11.308015] nf_conntrack version 0.5.0 (2048 buckets, 8192 max)
19:55:56.698 -> [   11.430126] usbcore: registered new interface driver ums-alauda
19:55:56.698 -> [   11.452611] usbcore: registered new interface driver ums-cypress
19:55:56.698 -> [   11.472661] usbcore: registered new interface driver ums-datafab
19:55:56.698 -> [   11.480338] usbcore: registered new interface driver ums-freecom
19:55:56.698 -> [   11.492668] usbcore: registered new interface driver ums-isd200
19:55:57.414 -> [   11.501218] urngd: v1.0.2 started.
19:55:57.414 -> [   11.506033] usbcore: registered new interface driver ums-jumpshot
19:55:57.414 -> [   11.522721] usbcore: registered new interface driver ums-karma
19:55:57.414 -> [   11.543035] usbcore: registered new interface driver ums-sddr09
19:55:57.414 -> [   11.562716] usbcore: registered new interface driver ums-sddr55
19:55:57.414 -> [   11.570613] usbcore: registered new interface driver ums-usbat
19:55:57.414 -> [   11.613228] xt_time: kernel timezone is -0000
19:55:57.414 -> [   11.818153] usbcore: registered new interface driver mt76x2u
19:55:57.414 -> [   11.838010] random: crng init done
19:55:57.414 -> [   11.841470] random: 6 urandom warning(s) missed due to ratelimiting
19:55:57.414 -> [   11.852616] PPP generic driver version 2.4.2
19:55:57.414 -> [   11.862370] NET: Registered protocol family 24
19:55:57.414 -> [   11.888166] usbcore: registered new interface driver rtl8187
19:55:57.414 -> [   11.953901] pci 0000:01:00.0: using irq 41 for pin 1
19:55:57.414 -> [   11.958946] ath10k 4.19 driver, optimized for CT firmware, probing pci device: 0x3c.
19:55:57.414 -> [   11.982242] PCI: Enabling device 0000:01:00.0 (0000 -> 0002)
19:55:57.414 -> [   11.988151] ath10k_pci 0000:01:00.0: pci irq legacy oper_irq_mode 1 irq_mode 0 reset_mode 0
19:55:57.841 -> [   12.301688] ath10k_pci 0000:01:00.0: Direct firmware load for ath10k/fwcfg-pci-0000:01:00.0.txt failed with error -2
19:55:57.841 -> [   12.312394] ath10k_pci 0000:01:00.0: Falling back to user helper
19:55:57.841 -> [   12.540656] firmware ath10k!fwcfg-pci-0000:01:00.0.txt: firmware_loading_store: map pages failed
19:55:57.841 -> [   12.549834] ath10k_pci 0000:01:00.0: Direct firmware load for ath10k/pre-cal-pci-0000:01:00.0.bin failed with error -2
19:55:57.841 -> [   12.560710] ath10k_pci 0000:01:00.0: Falling back to user helper
19:55:58.060 -> [   12.856570] firmware ath10k!pre-cal-pci-0000:01:00.0.bin: firmware_loading_store: map pages failed
19:55:58.060 -> [   12.865952] ath10k_pci 0000:01:00.0: Direct firmware load for ath10k/cal-pci-0000:01:00.0.bin failed with error -2
19:55:58.126 -> [   12.876481] ath10k_pci 0000:01:00.0: Falling back to user helper
19:55:58.455 -> [   13.262009] ath10k_pci 0000:01:00.0: Direct firmware load for ath10k/QCA988X/hw2.0/ct-firmware-5.bin failed with error -2
19:55:58.493 -> [   13.273166] ath10k_pci 0000:01:00.0: Falling back to user helper
19:55:58.604 -> [   13.411795] firmware ath10k!QCA988X!hw2.0!ct-firmware-5.bin: firmware_loading_store: map pages failed
19:55:58.642 -> [   13.421385] ath10k_pci 0000:01:00.0: Direct firmware load for ath10k/QCA988X/hw2.0/ct-firmware-2.bin failed with error -2
19:55:58.642 -> [   13.432534] ath10k_pci 0000:01:00.0: Falling back to user helper
19:55:58.793 -> [   13.575218] firmware ath10k!QCA988X!hw2.0!ct-firmware-2.bin: firmware_loading_store: map pages failed
19:55:58.793 -> [   13.584863] ath10k_pci 0000:01:00.0: Direct firmware load for ath10k/QCA988X/hw2.0/firmware-6.bin failed with error -2
19:55:58.793 -> [   13.595742] ath10k_pci 0000:01:00.0: Falling back to user helper
19:55:58.940 -> [   13.737071] firmware ath10k!QCA988X!hw2.0!firmware-6.bin: firmware_loading_store: map pages failed
19:55:58.940 -> [   13.746429] ath10k_pci 0000:01:00.0: Direct firmware load for ath10k/QCA988X/hw2.0/firmware-5.bin failed with error -2
19:55:58.978 -> [   13.757308] ath10k_pci 0000:01:00.0: Falling back to user helper
19:55:59.092 -> [   13.899674] firmware ath10k!QCA988X!hw2.0!firmware-5.bin: firmware_loading_store: map pages failed
19:55:59.130 -> [   13.909031] ath10k_pci 0000:01:00.0: Direct firmware load for ath10k/QCA988X/hw2.0/firmware-4.bin failed with error -2
19:55:59.130 -> [   13.919915] ath10k_pci 0000:01:00.0: Falling back to user helper
19:55:59.374 -> [   14.062128] firmware ath10k!QCA988X!hw2.0!firmware-4.bin: firmware_loading_store: map pages failed
19:55:59.374 -> [   14.071444] ath10k_pci 0000:01:00.0: Direct firmware load for ath10k/QCA988X/hw2.0/firmware-3.bin failed with error -2
19:55:59.374 -> [   14.082326] ath10k_pci 0000:01:00.0: Falling back to user helper
19:55:59.659 -> [   14.224668] firmware ath10k!QCA988X!hw2.0!firmware-3.bin: firmware_loading_store: map pages failed
19:55:59.959 -> [   14.533888] ath10k_pci 0000:01:00.0: qca988x hw2.0 target 0x4100016c chip_id 0x043202ff sub 0000:0000
19:55:59.959 -> [   14.543285] ath10k_pci 0000:01:00.0: kconfig debug 0 debugfs 1 tracing 0 dfs 1 testmode 0
19:55:59.959 -> [   14.555712] ath10k_pci 0000:01:00.0: firmware ver 10.1-ct-8x-__fW-022-538f0906 api 2 features wmi-10.x,has-wmi-mgmt-tx,mfp,txstatus-noack,wmi-10.x-CT,ratemask-CT,txrate-CT,get-temp-CT,tx-rc-CT,cust-stats-CT,retry-gt2-CT,txrate2-CT,beacon-cb-CT,wmi-block-ack-CT crc32 e1c91a74
19:55:59.959 -> [   14.608115] ath10k_pci 0000:01:00.0: Direct firmware load for ath10k/QCA988X/hw2.0/board-2.bin failed with error -2
19:55:59.959 -> [   14.618728] ath10k_pci 0000:01:00.0: Falling back to user helper
19:55:59.959 -> [   14.692674] firmware ath10k!QCA988X!hw2.0!board-2.bin: firmware_loading_store: map pages failed
19:55:59.959 -> [   14.712350] ath10k_pci 0000:01:00.0: board_file api 1 bmi_id N/A crc32 bebc7c08
19:56:00.883 -> [   15.646395] ath10k_pci 0000:01:00.0: 10.1 wmi init: vdevs: 16  peers: 127  tid: 256
19:56:00.883 -> [   15.663871] ath10k_pci 0000:01:00.0: wmi print 'P 128 V 8 T 410'
19:56:00.883 -> [   15.670159] ath10k_pci 0000:01:00.0: wmi print 'msdu-desc: 1424  sw-crypt: 0 ct-sta: 0'
19:56:00.883 -> [   15.678309] ath10k_pci 0000:01:00.0: wmi print 'alloc rem: 20984 iram: 25656'
19:56:01.274 -> [   15.732185] ath10k_pci 0000:01:00.0: htt-ver 2.1 wmi-op 2 htt-op 2 cal file max-sta 128 raw 0 hwcrypto 1
19:56:01.274 -> [   15.746456] ath10k_pci 0000:01:00.0: NOTE:  Firmware DBGLOG output disabled in debug_mask: 0x10000000
19:56:01.635 -> [   16.169591] usb 1-1: reset high-speed USB device number 2 using ehci-platform
19:56:01.942 -> [   16.521265] ieee80211 phy1: rt2x00_set_rt: Info - RT chipset 5390, rev 0502 detected
19:56:02.359 -> [   16.751640] ieee80211 phy1: rt2x00_set_rf: Info - RF chipset 5370 detected
19:56:02.359 -> [   16.769510] usbcore: registered new interface driver rt2800usb
19:56:02.359 -> [   16.822695] ieee80211 phy2: Atheros AR9550 Rev:0 mem=0xb8100000, irq=47
19:56:02.359 -> [   16.871917] kmodloader: done loading kernel modules from /etc/modules.d/*
19:56:15.100 -> [   29.888447] eth1: link up (1000Mbps/Full duplex)
19:56:15.100 -> [   29.893245] IPv6: ADDRCONF(NETDEV_UP): eth1: link is not ready
19:56:15.471 -> [   29.979282] br-lan: port 1(eth1.1) entered blocking state
19:56:15.471 -> [   29.984813] br-lan: port 1(eth1.1) entered disabled state
19:56:15.471 -> [   29.990513] device eth1.1 entered promiscuous mode
19:56:15.471 -> [   29.995419] device eth1 entered promiscuous mode
19:56:15.471 -> [   30.102524] br-lan: port 1(eth1.1) entered blocking state
19:56:15.471 -> [   30.108012] br-lan: port 1(eth1.1) entered forwarding state
19:56:15.471 -> [   30.157328] IPv6: ADDRCONF(NETDEV_UP): eth0: link is not ready
19:56:15.471 -> [   30.188446] IPv6: ADDRCONF(NETDEV_UP): eth0.2: link is not ready
19:56:15.471 -> [   30.206327] IPv6: ADDRCONF(NETDEV_CHANGE): eth1: link becomes ready
19:56:16.803 -> [   31.607097] eth0: link up (1000Mbps/Full duplex)
19:56:16.936 -> [   31.631621] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
19:56:16.936 -> [   31.650134] IPv6: ADDRCONF(NETDEV_CHANGE): eth0.2: link becomes ready
19:56:57.316 -> [   72.117386] jffs2_scan_eraseblock(): End of filesystem marker found at 0x0
19:56:57.397 -> [   72.131718] jffs2_build_filesystem(): unlocking the mtd device... 
19:56:57.397 -> [   72.131725] done.
19:57:06.844 -> [   72.139937] jffs2_build_filesystem(): erasing all blocks after the end marker... 
19:57:06.844 -> [   81.640062] done.
19:57:06.844 -> [   81.649664] jffs2: notice: (2438) jffs2_build_xattr_subsystem: complete building xattr subsystem, 0 of xdatum (0 unchecked, 0 orphan) and 0 of xref (0 dead, 0 orphan) found.
19:57:07.399 -> [   82.079070] overlayfs: upper fs does not support tmpfile.
19:57:11.170 -> [   85.980597] ieee80211 phy1: rt2x00lib_request_firmware: Info - Loading firmware file 'rt2870.bin'
19:57:11.640 -> [   86.435959] ieee80211 phy1: rt2x00lib_request_firmware: Info - Firmware detected - version: 0.36
19:57:12.110 -> [   86.877727] IPv6: ADDRCONF(NETDEV_UP): wlan1: link is not ready
19:57:12.505 -> [   87.284435] IPv6: ADDRCONF(NETDEV_UP): wlan2: link is not ready
19:57:15.347 -> [   90.144187] ath10k_pci 0000:01:00.0: 10.1 wmi init: vdevs: 16  peers: 127  tid: 256
19:57:15.498 -> [   90.161746] ath10k_pci 0000:01:00.0: wmi print 'P 128 V 8 T 410'
19:57:15.498 -> [   90.168051] ath10k_pci 0000:01:00.0: wmi print 'msdu-desc: 1424  sw-crypt: 0 ct-sta: 0'
19:57:15.498 -> [   90.176217] ath10k_pci 0000:01:00.0: wmi print 'alloc rem: 20984 iram: 25656'
19:57:15.498 -> [   90.245069] ath10k_pci 0000:01:00.0: pdev param 0 not supported by firmware
19:57:15.498 -> [   90.260436] IPv6: ADDRCONF(NETDEV_UP): wlan0: link is not ready
19:57:15.498 -> [   90.284725] br-lan: port 2(wlan0) entered blocking state
19:57:15.498 -> [   90.290230] br-lan: port 2(wlan0) entered disabled state
19:57:15.498 -> [   90.295936] device wlan0 entered promiscuous mode
19:57:15.801 -> [   90.433745] IPv6: ADDRCONF(NETDEV_CHANGE): wlan0: link becomes ready
19:57:15.801 -> [   90.440549] br-lan: port 2(wlan0) entered blocking state
19:57:15.801 -> [   90.446004] br-lan: port 2(wlan0) entered forwarding state
19:57:23.996 -> [   98.795736] IPv6: ADDRCONF(NETDEV_UP): wlan1: link is not ready
19:57:24.100 -> [   98.905180] IPv6: ADDRCONF(NETDEV_UP): wlan2: link is not ready
19:57:27.925 -> [  102.711468] device wlan0 left promiscuous mode
19:57:27.925 -> [  102.716124] br-lan: port 2(wlan0) entered disabled state
19:57:29.534 -> [  104.330125] ath10k_pci 0000:01:00.0: 10.1 wmi init: vdevs: 16  peers: 127  tid: 256
19:57:29.534 -> [  104.347624] ath10k_pci 0000:01:00.0: wmi print 'P 128 V 8 T 410'
19:57:29.890 -> [  104.353976] ath10k_pci 0000:01:00.0: wmi print 'msdu-desc: 1424  sw-crypt: 0 ct-sta: 0'
19:57:29.890 -> [  104.362132] ath10k_pci 0000:01:00.0: wmi print 'alloc rem: 20984 iram: 25656'
19:57:29.890 -> [  104.430568] ath10k_pci 0000:01:00.0: pdev param 0 not supported by firmware
19:57:29.890 -> [  104.445934] IPv6: ADDRCONF(NETDEV_UP): wlan0: link is not ready
19:57:29.890 -> [  104.455553] br-lan: port 2(wlan0) entered blocking state
19:57:29.890 -> [  104.461042] br-lan: port 2(wlan0) entered disabled state
19:57:29.890 -> [  104.466734] device wlan0 entered promiscuous mode
19:57:29.890 -> [  104.585747] IPv6: ADDRCONF(NETDEV_CHANGE): wlan0: link becomes ready
19:57:29.890 -> [  104.592377] br-lan: port 2(wlan0) entered blocking state
19:57:29.890 -> [  104.597773] br-lan: port 2(wlan0) entered forwarding state

```

**Comment by xchwarze** [comment]
Hi!
This is already fixed in the 2.8.5 version that was released last week

**Comment by xchwarze** [comment]
The fix is already tested in TL-WR1043ND v4
